zrml/rikiddo/src/mock.rs

```diff
@@ -148,6 +149,9 @@ impl ExtBuilder {
     pub fn build(self) -> sp_io::TestExternalities {
         let mut t = frame_system::GenesisConfig::default().build_storage::<Runtime>().unwrap();
 
+        // see the logs in tests when using `cargo test -- --nocapture`
```

Comment:
We should mention that `RUST_LOG=debug cargo test` is the required call to see infos/warnings.

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/1205/commits/66e69565728c008794799cb2b6e5d6ea00ec6dff